fastlane/lib/fastlane/swift_fastlane_function.rb

```diff
@@ -31,7 +31,7 @@ def initialize(action_name: nil, action_description: nil, action_details: nil, k
 
       # rubocop:disable LineLength
       # class instance?
-      @reserved_words = %w[associativity break case catch class continue convenience default deinit didSet do else enum extension fallthrough false final for func get guard if in infix init inout internal lazy let mutating nil operator override postfix precedence prefix private public repeat required return self set static struct subscript super switch throws true try var weak where while willSet].to_set
+      @reserved_words = %w[associativity break case catch class continue convenience default deinit didSet do else enum extension fallthrough false final for func guard if in infix init inout internal lazy let mutating nil operator override precedence private public repeat required return self static struct subscript super switch throws true try var weak where while willSet].to_set
```

Comment:
What is this about?

Reply:
This are not required reserved words in Swift 4.0. There was a time that they were in Swift 3 where you had to backtick them, but not in the current project version